pkg/test/csrctrl/controllers/start_csrctrl.go

```diff
@@ -83,19 +75,36 @@ func RunCSRController(signerNames string, appendRootCert bool, config *rest.Conf
 			log.Infof("Unable to read root cert for signer [%s], error: %v", signerName, sErr)
 			os.Exit(-1)
 		}
-		rootCertsForSigner := &SignerRootCert{
+		rootCertsForSigner := SignerRootCert{
 			Signer:   signerName,
 			Rootcert: string(rootCert),
 		}
-		certChan <- rootCertsForSigner
+		rootCertSignerArr = append(rootCertSignerArr, rootCertsForSigner)
 	}
 
+	for _, cluster := range clusters {
+		mgr, err := ctrl.NewManager(cluster.RESTConfig(), ctrl.Options{
+			Scheme: scheme,
+			// disabel the metric server to avoid the port conflicting
+			MetricsBindAddress: "0",
+		})
+		if err != nil {
+			log.Infof("Unable to start manager error: %v", err)
+			os.Exit(-1)
+		}
+		go runManager(mgr, arrSigners, signersMap, appendRootCert, c)
+	}
```

Comment:
cc @zhlsunshine – not sure the reason for the original design with channel, PTAL